tools/manage_clvm.py

```diff
@@ -37,21 +37,60 @@
 top_levels = {"chia"}
 
 
+class ManageClvmError(Exception):
+    pass
+
+
 class CacheEntry(typing.TypedDict):
     clvm: str
     hex: str
     hash: str
 
 
 CacheEntries = typing.Dict[str, CacheEntry]
+CacheVersion = typing.List[int]
+current_cache_version: CacheVersion = [1]
+
+
+class CacheVersionError(ManageClvmError):
+    pass
+
+
+class NoCacheVersionError(CacheVersionError):
```

Comment:
We could get rid of this and it's usage by using `WrongCacheVersionError(None, current_cache_version)` if there is no version since `found_version` is an `object`, maybe.